### PR TITLE
Meenu/Updated close icon as button

### PR DESCRIPTION
**Pull request by meenakshi-deriv**

### CODE REVIEW

File: lib/components/Modal/base/index.tsx [hunk 1]
```diff
@@ -145,10 +145,12 @@ export const Modal = ({
...
                     )
                 ) : (
-                    <LabelPairedXmarkMdBoldIcon
+                    <button
                         className="quill-modal__close-icon"
                         onClick={toggleHandler}
-                    />
+                    >
+                        <LabelPairedXmarkMdBoldIcon fill="var(--component-textIcon-normal-prominent)" />
+                    </button>
```
Comment (jia-deriv) [inline lib/components/Modal/base/index.tsx:153]: If we use our quill button component will have the padding I guess.

Comment (meenakshi-deriv) [reply lib/components/Modal/base/index.tsx:153]: we don't have this variant in quill button and it all comes with a border radius or underline. so i used custom button

File: lib/components/Modal/modal.scss [hunk 2]
```diff
@@ -82,7 +82,11 @@
     &__close-icon {
         position: absolute;
         cursor: pointer;
-        top: var(--core-spacing-1600);
+        top: var(--core-spacing-1400);
+        width: var(--core-spacing-2400);
+        padding: var(--core-spacing-600) var(--core-spacing-800);
+        justify-content: center;
+        display: flex;
         right: var(--core-spacing-1600);
```
Comment (jia-deriv) [inline lib/components/Modal/modal.scss:90]:
```suggestion
        right: var(--core-spacing-800);
```

File: lib/components/Modal/modal.scss [hunk 3]
```diff
@@ -82,7 +82,11 @@
     &__close-icon {
         position: absolute;
         cursor: pointer;
-        top: var(--core-spacing-1600);
+        top: var(--core-spacing-1400);
```
Comment (jia-deriv) [inline lib/components/Modal/modal.scss:85]:
```suggestion
        top: var(--core-spacing-800);
```

image

**Review by jia-deriv — APPROVED** [review]

**Review by prince-deriv — APPROVED** [review]

**Pull request merged**